Makefile

```diff
@@ -525,7 +525,7 @@ localnet_up: ## Starts up a k8s LocalNet with all necessary dependencies (tl;dr
 
 .PHONY: localnet_client_debug
 localnet_client_debug: ## Opens a `client debug` cli to interact with blockchain (e.g. change pacemaker mode, reset to genesis, etc). Though the node binary updates automatiacally on every code change (i.e. hot reloads), if client is already open you need to re-run this command to execute freshly compiled binary.
-	kubectl exec -it deploy/dev-cli-client --container pocket -- p1 debug --remote_cli_url http://pocket-validators:50832
+	kubectl exec -it deploy/dev-cli-client --container pocket -- p1 DebugUI --remote_cli_url http://pocket-validators:50832
```

Comment:
Is there a reason that we're using a different remote CLI URL here and/or should we prefer the ENV var for this case?